thriftbp/client_pool.go

```diff
@@ -228,6 +228,12 @@ type ClientPoolConfig struct {
 	//
 	// Optional. If empty, no "thrift-hostname" header will be sent.
 	ThriftHostnameHeader string `yaml:"thriftHostnameHeader"`
+
+	// Uses zlib transform in thrift THeader connections between client and
```

Comment:
is THeader just for headers, or for the whole connection?  `UseZlib` sounds like this is for the whole connection, but the docs make it sound like it's just for headers...

so maybe
* rename to `UseZlibForHeaders`, or
* update the comment to specify that this is for the whole connection

?

Reply:
`THeader` is a thrift TProtocol/TTransport (it's just the only TProtocol/TTransport that supports headers, thus the name)

so it's for the whole connection.